yt_dlp/extractor/ceskatelevize.py

```diff
@@ -133,29 +131,9 @@ def _real_extract(self, url):
         if any(not_found in webpage for not_found in ('Neplatný parametr pro videopřehrávač', 'IDEC nebyl nalezen', )):
             raise ExtractorError('no video with IDEC available', video_id=idec, expected=True)
 
-        type_ = None
-        episode_id = None
-
-        playlist = self._parse_json(
-            self._search_regex(
-                r'getPlaylistUrl\(\[({.+?})\]', webpage, 'playlist',
-                default='{}'), playlist_id)
-        if playlist:
-            type_ = playlist.get('type')
-            episode_id = playlist.get('id')
-
-        if not type_:
-            type_ = self._html_search_regex(
-                r'getPlaylistUrl\(\[\{"type":"(.+?)","id":".+?"\}\],',
-                webpage, 'type')
-        if not episode_id:
-            episode_id = self._html_search_regex(
-                r'getPlaylistUrl\(\[\{"type":".+?","id":"(.+?)"\}\],',
-                webpage, 'episode_id')
-
         data = {
-            'playlist[0][type]': type_,
-            'playlist[0][id]': episode_id,
+            'playlist[0][type]': 'episode',
```

Comment:
Are you sure there are no other "type_"s now?

Reply:
Obviously I cannot be 100% sure. But I checked different types of entries (a movie, a show episode and a live broadcast) and the browser used "episode" for all of them. So in the end I decided to submit it like this and see if someone finds a case where it does not work correctly (which can be addressed when/if that happens).

Reply:
With the updated branch (commit b73a061d3150), all selftests work now, except for one where the URL is no longer valid and without further information, I couldn't find what should the updated URL look like. I also set `only_matching` for live broadcast tests as the framework insisted on setting `duration` key for the tests but that depends on currently broadcasted program; also, for CT1 channel the site no longer has stable item id and uses the playlist id of current program instead. But all live channels work unless there is a break, live broadcast is unavailable or it is DRM protected.